### PR TITLE
Documentation to add EXTENT in the LAYER

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/administrator/mapfile.rst [hunk 1]
```diff
@@ -400,6 +400,14 @@ Mapserver allows different forms of legends.
     END
 
 
+Performance imrovement
```
Comment (asaunier) [inline doc/administrator/mapfile.rst]: improvement

File: doc/administrator/mapfile.rst [hunk 2]
```diff
@@ -400,6 +400,14 @@ Mapserver allows different forms of legends.
...
+----------------------
+
+In the ``LAYER`` we can specify an ``EXTENT`` we can specify an extent to that can be a big
```
Comment (asaunier) [inline doc/administrator/mapfile.rst]: Adding an ``EXTENT`` parameter in the ``LAYER`` section may significantly improve the performances.

**Comment by sbrunner** [comment]
Thanks, updated :-)

File: doc/administrator/mapfile.rst [hunk 3]
```diff
@@ -400,6 +400,12 @@ Mapserver allows different forms of legends.
     END
 
 
+Performance improvement
+-----------------------
+
+Adding an ``EXTENT`` parameter in the ``LAYER`` section may significantly improve the performances
```
Comment (asaunier) [inline doc/administrator/mapfile.rst]: oops, maybe "adding to" is better than "adding in"?

**Comment by asaunier** [comment]
Looks good to me

**Pull request merged**